low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -60,6 +60,9 @@ def validate(self):
         """
         machine_id = Utility.get_machine_id()
 
+        # configure sspl-setup log dir before validation stage.
```

Comment:
well why not in init itself.
validate() should have validations alone and not any pre-requisite prep

Reply:
Everytime PostInstall  class object will be created, sspl-setup dir will be configured if put in init.

Reply:
check for dir existence and if missing, then only create. validate is not right place for such actions

Reply:
Addressed

Reply:
@cdeshmukh we need to configure sspl-setup log dir in validate stage only.
because while configuring it requires values in product name etc which are getting fetched in validation stage only.
And these values cannot be fetched in init stage because in sspl-setup, Conf index gets loaded in validation stage only
so after the validation of product name & setup then we can configure the setup-log dir